src/Microsoft.Health.Fhir.Core/Features/Search/Converters/BooleanNodeToTokenSearchValueTypeConverter.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Health.Fhir.Core.Features.Search.Converters
     public class BooleanNodeToTokenSearchValueTypeConverter : FhirNodeToSearchValueTypeConverter<TokenSearchValue>
     {
         public BooleanNodeToTokenSearchValueTypeConverter()
-            : base("boolean")
+            : base("boolean", "System.Boolean")
```

Comment:
TODO: Check other converter types

Reply:
`System.Boolean` was returned when evaluating `Patient.deceased.exists() and Patient.deceased != false`. This previously only returned `boolean` as the ITypedElement.InstanceType